src/app/pcpolicy/pcpolicy.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { DashboardService } from '../services/dashboard.service';
+
+@Component({
+  selector: 'app-pcpolicy',
+  templateUrl: './pcpolicy.component.html',
+  styleUrls: ['./pcpolicy.component.scss']
+})
+
+export class PcpolicyComponent implements OnInit {
+    email: String;
+    public position= 'col-md-10 col-md-offset-2';
```

Comment:
@yukie1412 I have invited you with write access already. Once you accept you will get write access.